### ISSUE TITLE
configuration error on macos, because of missing (needless) libbsd

### ISSUE THREAD

**Issue by hdatma** [graywolf/acme-client-portable#2]
```
> aclocal
...
> autoreconf -i
...
> autoreconf
...
./configure --prefix=/test --with-openssl=/opt/libressl
...
checking whether compiling and linking against OpenSSL works... yes
checking for pkg-config... /usr/local/bin/pkg-config
checking for bsd/bsd.h in /usr/local/bsd... no
checking for bsd/bsd.h in /usr/lib/bsd... no
checking for bsd/bsd.h in /usr/bsd... no
checking for bsd/bsd.h in /usr/pkg... no
checking for bsd/bsd.h in /usr/local... no
checking for bsd/bsd.h in /usr... no
checking whether compiling and linking against libbsd works... no
```
```
>uname -a
Darwin alice 17.7.0 Darwin Kernel Version 17.7.0: Sun Jun  2 20:31:42 PDT 2019; root:xnu-4570.71.46~1/RELEASE_X86_64 x86_64
```

**Comment by graywolf** [comment]
Sorry, I do not have macos device available to even if I tried to fix it, I would have no way to check it actually works. Feel free to provide patch.

**Comment by graywolf** [comment]
Should be resolved in 0.3.0.
